src/lib/cm/createCmTemplateFolder.ts

```
@@ -0,0 +1,131 @@
+            message: `File ${filePath} already exists. Overwrite? (y/N): `,
+            default: false
+        })
+        if (!answer) {
+            console.log(`Skipping overwrite of ${filePath}`)
+            return false
+        }
+    }
+    return true
+}
+
+const createCmTemplateFolder = async (name?: string, fullTemplate: boolean = false) => {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)